clerk/organizations.go

```diff
@@ -181,7 +181,7 @@ type ListOrganizationMembershipsParams struct {
 	Query          *string  `json:"query"`
 }
 
-type ΟrganizationMembershipPublicUserData struct {
+type OrganizationMembershipPublicUserData struct {
```

Comment:
❗ We can't rename those, as this is a breaking change. We will have to wait for the next major version

Reply:
I will revert the changes on this.